main/FatesConstantsMod.F90

```diff
@@ -34,6 +34,9 @@ module FatesConstantsMod
   integer, parameter, public :: n_anthro_disturbance_categories = 2
   integer, parameter, public :: primaryforest = 1
   integer, parameter, public :: secondaryforest = 2
+  integer, parameter, public :: secondary_age_threshold = 94 ! less than this value is young
```

Comment:
What does 'young' imply? That it won't be harvested as 'secondary' forest?

Reply:
this is the distinction between secondary old and secondary young, i.e. in the harvest rates `HARVEST_SH1` vs `HARVEST_SH2` and `HARVEST_SH3`.  It thus should effectively act as a rotation age for the secondary forest harvest.

Reply:
It also determines how to apply HARVEST_SH2 (young secondary) and HARVEST_SH3 (secondary non-forest), which presumably have less biomass than mature secondary (to which HARVEST_SH1 is applied). SH2 and SH3 are applied to the "young" patches, and SH1 is applied to the "mature" patches.

Reply:
this is compared against a real (secondary_age), recommend setting the parameter to a real as well

Reply:
```suggestion
  real(fates_r8), parameter, public :: secondary_age_threshold = 94._fates_r8 ! less than this value is young secondary forest
```